### ISSUE TITLE
Add a pre-built windows binary

### ISSUE THREAD

**Issue by mkatzNU** [keirf/disk-utilities#131]
It might not be a bad idea to have a windows binary available for download for this like you have for the greaseweazle software.

Thank you.

**Comment by kaunomedis** [comment]
Yeah, it could be fine. I managed to compile it when removed macro in util.h
      
      #define err(retval, ...)  fprintf(stderr, "Undefined error: %d\n", errno);
      /*
      #define err(retval, ...) do { \
          fprintf(stderr, __VA_ARGS__); \
          fprintf(stderr, "Undefined error: %d\n", errno); \
          exit(retval); \
      } while(0)
      */
